<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>clientWindow</class>
 <widget class="QMainWindow" name="clientWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>420</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>420</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>clientWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="serialEdit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>220</width>
      <height>32</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
   </widget>
   <widget class="QLabel" name="serialLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>125</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Serial number</string>
    </property>
   </widget>
   <widget class="QLabel" name="ipLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>125</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>IP</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ipEdit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>120</y>
      <width>220</width>
      <height>32</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
   </widget>
   <widget class="QLabel" name="maskLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>125</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Netmask</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="maskEdit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>170</y>
      <width>220</width>
      <height>32</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
   </widget>
   <widget class="QLabel" name="gatewayLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>125</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Gateway</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="gatewayEdit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>220</y>
      <width>220</width>
      <height>32</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
   </widget>
   <widget class="QLabel" name="nameLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>125</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameEdit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>270</y>
      <width>220</width>
      <height>32</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
   </widget>
   <widget class="QLabel" name="macLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>125</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>MAC</string>
    </property>
   </widget>
   <widget class="QLabel" name="macDisplay">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>340</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>MAC</string>
    </property>
   </widget>
   <widget class="QLabel" name="typeDisplay">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>370</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Type</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>420</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
